src/main/java/reposense/git/exception/GitCloneException.java

```diff
@@ -1,6 +1,8 @@
 package reposense.git.exception;
 
-
+/**
+ * Represents the exception of git clone.
```

Comment:
Can rephrase this to follow format of the other `Exception` (e.g. `Signals that a commit is not found` in `CommitNotFoundException`)